extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigationOnSiteNotifications_index.php

```diff
@@ -0,0 +1,19 @@
+<div id="onSiteNotificationsEntryPoint"
+     class="wds-global-navigation__notifications-menu wds-global-navigation__notifications-menu wds-dropdown notifications-container">
+	<div class="wds-dropdown__toggle wds-global-navigation__dropdown-toggle"
+	     title="<?= wfMessage( 'global-navigation-notifications-title' )->escaped() ?>">
+		<div class="bubbles">
```

Comment:
Can we get rid of this container / class? It was used in the Global Navigation Notifications because we didn't want to refactor the legacy code too much. I doubt it's needed here.